01_html_basics/starter_code/cookie_recipe/index.html

```diff
@@ -14,27 +14,75 @@
 - The color for the text reading "Nutrition Information" is #887
 - The link color is #900
 
-Bonus: The border on the page and around the image is black
+Bonus: The border on the page and around the image is black -->
 
-The Cookie Recipe
-=================
+<!-- The Cookie Recipe -->
+
+<!-- ================= -->
+
+
+<h1>The Best Chocolate Chip Cookies</h1>
+
+<br>
+
+<img src="cookies.jpg" ALT="Chocolate chip cookie">
```

Comment:
Not _invalid_ technically, but the most widely accepted convention is to write html tags and their attributes in all lower-case, so I'd just get in the habit of leaving your attributes, like `ALT` in lower-case :)